packages/content/scripts/compile-mdx.jsx

```
@@ -0,0 +1,190 @@
+import fs from 'fs/promises'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)